changelog.txt

```diff
@@ -2,6 +2,8 @@ OPENSEADRAGON CHANGELOG
 =======================
 
 2.0.1: (in progress)
+    * BREAKING CHANGE: the tile does not hold a reference to its image anymore. Only the tile cache keep a reference to images.
```

Comment:
While you're in here you might as well also add the line for the cool new feature you're adding :)